cranelift/fuzzgen/src/function_generator.rs

```diff
@@ -1224,16 +1259,19 @@ where
     ///
     /// `min_size`: Controls the amount of space that the address should have.This is not
     /// guaranteed to be respected
```

Comment:
Pre-existing, but I believe this comment is wrong at this point: there are always `min_size` valid bytes at the base+offset returned by this function. If I remember correctly the comment was because we used to sometimes deliberately generate invalid `heap_addr` instructions, right?

Reply:
Yeah I think so, but we probably don't do that anymore. I'm going to remove it.